IBF-Typhoon-model/lib/pipeline.py

```diff
@@ -0,0 +1,314 @@
+#!/bin/sh
+
+# -*- coding: utf-8 -*-
+"""
+Created on Sat Oct 31 16:01:00 2020
+
+@author: ATeklesadik
+"""
+import sys
+import os
+from datetime import datetime, timedelta
+from sys import platform
+import subprocess
+import logging
+import traceback
+
+import pandas as pd
+from pybufrkit.decoder import Decoder
+import numpy as np
+from geopandas.tools import sjoin
+import geopandas as gpd
+import click
+
+# Set up logger
+logging.root.handlers = []
+logging.basicConfig(format='%(asctime)s : %(levelname)s : %(message)s', level=logging.DEBUG, filename='ex.log')
+# set up logging to console
+console = logging.StreamHandler()
+console.setLevel(logging.INFO)
+# set a format which is simpler for console use
+formatter = logging.Formatter('%(asctime)s : %(levelname)s : %(message)s')
+console.setFormatter(formatter)
+logging.getLogger("").addHandler(console)
```

Comment:
Why did you remove the following?
```python
# Stop some overly verbose packages
for log_name, log_obj in logging.Logger.manager.loggerDict.items():
    if log_name != '<module name>':
        logging.getLogger(log_name).setLevel(max(logging.WARNING, level))
```
I mainly added it because `pybufrkit` outputs a bunch of useless messages when processing the typhoon data (and it's difficult to set the level individually because it appears in the logger as a bunch of modules), although this could be done if you think my approach was too aggressive